app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt

```diff
@@ -192,6 +200,22 @@ class PinPasswordActivityTest {
     }
   }
 
+  @Test
+  fun testPinPasswordActivity_hasCorrectActivityLabel() {
+    activityTestRule.launchActivity(
+      PinPasswordActivity.createPinPasswordActivityIntent(
+        context = context,
```

Comment:
The test case is failing because of this line, suggested using this:
`context = ApplicationProvider.getApplicationContext(),`

Also, always make sure to run the tests after the commit.

Reply:
Is it so? I ran this test function before committing and it passed. 
I ran it today again
![image](https://user-images.githubusercontent.com/46752548/113386614-410b9900-93a8-11eb-83c8-15fd5f737658.png)
it is passing.